webapp/src/Controller/API/ProblemController.php

```diff
@@ -181,6 +181,11 @@ public function listAction(Request $request): Response
      *                 property="problem",
      *                 description="Optional: problem id to update.",
      *                 type="string"
+     *             ),
+     *             @OA\Property(
+     *                 property="delete_old_data",
+     *                 description="Optional: whether to delete old (existing) data before importing into an existing problem.",
```

Comment:
Why do you introduce this? I think this is might be part of an older commit in our main branch.